charts/hivemq-platform/values.yaml

```diff
@@ -88,17 +88,41 @@ restApi:
   # Enables or disables authentication and authorization.
   authEnabled: false
 
```

Comment:
In general we should align to the documentation here. 
The "main" license is just called "HiveMQ license", so I guess we should stick to that. I will try a set of suggestions for this. Let me know what you think.

Reply:
Yeah, I agree in most of the comments. Have a look.